src/PatchOrchestrationApplication/CoordinatorService/src/PackageRoot/ServiceManifest.xml

```diff
@@ -21,7 +21,7 @@
 
   <!-- Config package is the contents of the Config directoy under PackageRoot that contains an 
        independently-updateable and versioned set of custom configuration settings for your service. -->
-  <ConfigPackage Name="Config" Version="1.1.0" />
+  <ConfigPackage Name="Config" Version="1.1.1" />
```

Comment:
>1 [](start = 44, length = 1)

Have we changed anything in config? Should we make it sync with the overall version like at other places

Reply:
yes

Reply:
I will resolve this in the last PR of this version release, I just need intermediate versions because I will test the application through upgrades.